hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/FairScheduler.md

```diff
@@ -104,7 +104,7 @@ The allocation file must be in XML format. The format contains five types of ele
 
     * **maxAMShare**: limit the fraction of the queue's fair share that can be used to run application masters. This property can only be used for leaf queues. For example, if set to 1.0f, then AMs in the leaf queue can take up to 100% of both the memory and CPU fair share. The value of -1.0f will disable this feature and the amShare will not be checked. The default value is 0.5f.
 
-    * **weight**: to share the cluster non-proportionally with other queues. Weights default to 1, and a queue with weight 2 should receive approximately twice as many resources as a queue with the default weight.
+    * **weight**: to share the cluster non-proportionally with other queues. Weights default to 1, and a queue with weight 2 should receive approximately twice as many resources as a queue with the default weight. A queue with weight 0 should receive resources after all non-zero weight queues have received resources. If all weights are equal to 0, a queue with less resource usage should receive resources.
```

Comment:
"queue with less resource usage should receive resources" sounds assuming "fair" schedulingPolicy. You can say "queues get equal share" or "same to queues with same non-zero weight"?